README.md

```diff
@@ -65,7 +65,7 @@ Replace `<GITHUB_TOKEN>`, `<OPENAI_TOKEN>`, `<OWNER>`, `<REPO>`, and `<PR_NUMBER
```

Comment:
It would be useful to have some comments explaining the purpose and functionality of the code.